From 98270129df6cb21aedcd90899e4f0223ff1c57f1 Mon Sep 17 00:00:00 2001
From: Stefan Haller <stefan@haller-berlin.de>
Date: Tue, 13 Oct 2020 11:36:02 +0200
Subject: [PATCH 03/19] git-gui/gitk: Delete the code to bring app to the front

With a modern enough Tcl/Tk this no longer seems to be needed.
---
 git-gui/git-gui.sh | 14 --------------
 gitk-git/gitk      |  9 ---------
 2 files changed, 23 deletions(-)

diff --git a/git-gui/git-gui.sh b/git-gui/git-gui.sh
index d0c56dc4f5..bd9c97061f 100755
--- a/git-gui/git-gui.sh
+++ b/git-gui/git-gui.sh
@@ -264,20 +264,6 @@ proc strcat {args} {
 ::msgcat::mcload $oguimsg
 unset oguimsg
 
-######################################################################
-##
-## On Mac, bring the current Wish process window to front
-
-if {[tk windowingsystem] eq "aqua"} {
-	catch {
-		exec osascript -e [format {
-			tell application "System Events"
-				set frontmost of processes whose unix id is %d to true
-			end tell
-		} [pid]]
-	}
-}
-
 ######################################################################
 ##
 ## read only globals
diff --git a/gitk-git/gitk b/gitk-git/gitk
index 58ceabc15d..5567b2aa08 100755
--- a/gitk-git/gitk
+++ b/gitk-git/gitk
@@ -12509,15 +12509,6 @@ if {[catch {package require Tk 8.4} err]} {
     exit 1
 }
 
-# on OSX bring the current Wish process window to front
-if {[tk windowingsystem] eq "aqua"} {
-    exec osascript -e [format {
-        tell application "System Events"
-            set frontmost of processes whose unix id is %d to true
-        end tell
-    } [pid] ]
-}
-
 # Unset GIT_TRACE var if set
 if { [info exists ::env(GIT_TRACE)] } {
     unset ::env(GIT_TRACE)
-- 
2.48.1

